R/pkg/R/functions.R

```diff
@@ -209,6 +209,7 @@ NULL
 #' head(select(tmp, array_max(tmp$v1), array_min(tmp$v1)))
 #' head(select(tmp, array_position(tmp$v1, 21)))
 #' head(select(tmp, flatten(tmp$v1)))
+#' head(select(tmp, reverse(tmp$v1)))
```

Comment:
Also add `concat` here?

Reply:
let's merge into an existing line - we don't need one line per function; don't want these too be too long each line but also not too many lines either